docker/post-install

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+
+dnf install -y /tmp/rpms/*.rpm
+
+# Be nice for docker exec and link SPDK scripts|binaries under common PATH
+# location like /usr/bin.
+ln -sf $(ls -1dpA /usr/local/bin/* | grep -v "/$") /usr/bin
+ln -s /usr/libexec/spdk/scripts/rpc.py /usr/bin
+ln -s /usr/libexec/spdk/scripts/rpc_http_proxy.py /usr/bin
+ln -s /usr/libexec/spdk/scripts/setup.sh /usr/bin
+ln -s /usr/libexec/spdk/include/spdk /usr/include
+ln -s /usr/libexec/spdk/scripts/ /usr
```

Comment:
This should rather be added to the spdk.spec file, or if, as stated, this is just a 'hack' only for containers, then add `PATH+=":/usr/libexec/spdk/scripts/" to the Dockerfile envs.